### DISCUSSION TITLE
How does CefSharp.Wpf.ChromiumWebBrowser preload a HTML page?

### DISCUSSION THREAD

**Discussion by wlh21cn** [cefsharp/CefSharp#3465] [Q&A]
I use CefSharp.Wpf.ChromiumWebBrowser for my app, my app has several pages, only one page is active page. Every page has one or several ChromiumWebBrowser instances. I want ChromiumWebBrowser instances in all pages can preload HTML with URLs, so that the page is ready after active the page. Now only the ChromiumWebBrowser intances in first page can load HTML, the instances in other pages cannot load HTML, even after I click and show the page.

I searched some solution, such as link below. It says call browser.CreateControl(), but CreateControl is only for CefShap.WinForm.ChromiumWebBrowser, CefSharp.Wpf.ChromiumWebBrowser does not have method CreateControl.
https://github.com/cefsharp/CefSharp/issues/1406

Does anyone know how to solve this issue? Thanks in advance.
----
Neal

**Comment by amaitland** [comment]
You can use the constructor at http://cefsharp.github.io/api/88.2.x/html/M_CefSharp_Wpf_ChromiumWebBrowser__ctor_2.htm

See also https://stackoverflow.com/questions/2193166/how-do-i-prerender-the-controls-on-a-tabitem-in-wpf/10210889
